public/src/components/Groups.tsx

```diff
@@ -33,19 +35,38 @@ const Groups = (): JSX.Element => {
         return true
     }
 
-    const updateGroupStatus = (group: Group, status: Group_GroupStatus) => {
-        actions.updateGroupStatus({ group, status })
-    }
-
     const GroupButtons = ({ group }: { group: Group }) => {
         const buttons: JSX.Element[] = []
         if (isPendingGroup(group)) {
-            buttons.push(<span onClick={() => updateGroupStatus(group, Group_GroupStatus.APPROVED)} className="badge badge-primary clickable">Approve</span>)
+            buttons.push(
+                <DynamicButton
+                    onClick={() => actions.updateGroupStatus({ group, status: Group_GroupStatus.APPROVED })}
+                    color={Color.BLUE}
+                    type={ButtonType.BADGE}
+                    text={"Approve"}
+                />
+            )
         }
-        buttons.push(<span className="badge badge-info clickable ml-2" onClick={() => actions.setActiveGroup(group)}>Edit</span>)
-        buttons.push(<span onClick={() => actions.deleteGroup(group)} className="badge badge-danger clickable ml-2">Delete</span>)
+        buttons.push(
+            <Button
```

Comment:
General comment for all buttons: Should we adopt a common "style" for all buttons that places fields in this order:

- text
- color
- type
- className
- onClick